utils/build.ps1

```diff
@@ -2172,6 +2172,21 @@ function Build-Sanitizers([Hashtable] $Platform) {
     }
 }
 
+function Build-Brotli([Hashtable] $Platform) {
+  Build-CMakeProject `
+    -Src $SourceCache\brotli `
+    -Bin "$BinaryCache\$($Platform.Triple)\brotli" `
```

Comment:
Please add this to the enumeration of targets that we build near the top and use the path computation helpers. That makes it easier to shuffle directories around if we need to.